resources/assets/js/components/ResizeShim.vue

```diff
@@ -30,11 +30,15 @@ export default {
 	},
 
 	mounted() {
```

Comment:
Unless sure that it will be i think its worth keeping in the if statement?

Reply:
Yes those if statements were put in because js complained in the console about ```this.$refs['hidden-frame'].contentWindow``` being null. This is in the case where the preview is replace with the error.